internal/pkg/planmanager/result.go

```diff
@@ -31,67 +29,17 @@ func (p *Plan) handleResult(change *Change) error {
 	// uninstall failed
 	if change.ActionName == statemanager.ActionUninstall && !change.Result.Succeeded {
 		state.Status = statemanager.StatusInstalled
-		log.Printf("=== plugin %s uninstall failed ===", change.Tool.Name)
-		// install or reinstall failed
+		log.Printf("Plugin %s uninstall failed.", change.Tool.Name)
```

Comment:
Log printing generally starts with lowercase

Reply:
At the moment, we have different formats, which will be improved in the next sprint. Closing this comment for now since we will improve it in the next sprint thoroughly.

Some discussion:

- Log messages are line-oriented, so they may be full sentences. For full sentences, the first letter should be uppercase and it should end with punctuation.
- Au-contraire, errors should start with lowercase letters.